cuda_core/cuda/core/experimental/_stream.py

```diff
@@ -71,18 +71,37 @@ def close(self):
                 self.owner = None
             self.handle = None
 
+    def __new__(self, *args, **kwargs):
+        raise RuntimeError(
+            "Stream objects cannot be instantiated directly. "
+            "Please use Device APIs (create_stream) or other Stream APIs (from_handle, wait)."
```

Comment:
`wait()` is not a constructor:
```suggestion
            "Please use Device APIs (create_stream) or other Stream APIs (from_handle)."
```

Reply:
Yes, thanks for catching this!

`wait` removed in commit 201d3fc17e42262c5aed965e7257edd6992f90c4

Your review went to "stale", could you please re-approve?